en-US/firefox/home-master.lang

```diff
@@ -0,0 +1,103 @@
+## NOTE: Demo http://www-demo4.allizom.org/firefox/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/
+
+
+# HTML page title
+;Protect your life online with Firefox and a family of privacy-first products
+Protect your life online with Firefox and a family of privacy-first products
+
+
+# HTML description
+;Firefox is more than a browser. Learn more about Firefox products that handle your data with respect and are built for privacy anywhere you go online.
+Firefox is more than a browser. Learn more about Firefox products that handle your data with respect and are built for privacy anywhere you go online.
+
+
+;The browser is just the beginning
+The browser is just the beginning
+
+
+;Meet our family of products
+Meet our family of products
+
+
+;Get 2,000+ trackers off your trail — including Facebook
+Get 2,000+ trackers off your trail — including Facebook
+
+
+;Know when hackers strike — and stay a step ahead
+Know when hackers strike — and stay a step ahead
+
+
+# CTA link
+;Start getting breach reports
+Start getting breach reports
+
+
+;Keep your passwords safe on every device
+Keep your passwords safe on every device
+
+
+# Lockwise is a product name that shouldn't be translated
+;Learn more about Lockwise
+Learn more about Lockwise
+
+
+# the strong tags around "respect" add a special underline, the underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+;Get the <strong>respect</strong> you deserve
+Get the <strong>respect</strong> you deserve
+
+
+;Every single Firefox product honors our Personal Data Promise: <strong>Take less. Keep it safe. No secrets.</strong>
+Every single Firefox product honors our Personal Data Promise: <strong>Take less. Keep it safe. No secrets.</strong>
+
+
+;Share large files without prying eyes
+Share large files without prying eyes
+
+
+# CTA link
+;Start sending files safely
+Start sending files safely
+
+
+;Trade clickbait for quality content
+Trade clickbait for quality content
+
+
+# Pocket is a product name that shouldn't be translated
+;Learn more about Pocket
+Learn more about Pocket
+
+
+;One login. All your devices. A family of products that respect your <strong>privacy</strong>.
```

Comment:
Do we need at comment for `privacy` too, tag no more than one word? Here is how [Firefox Privacy](https://transvision.mozfr.org/string/?entity=mozilla_org/download_button.lang:d78ffa8d9a0cf6277e22432270bc014d&repo=mozilla_org) is translated. Some have multiple words.